yogstation/code/modules/uplink/uplink_item.dm

```diff
@@ -60,6 +60,15 @@
 	item = /obj/item/flashlight //doesn't actually spawn a flashlight, but it needs an object to show up in the menu :^)
 	cost = 5
 	surplus = 0
+	exclude_modes = list(/datum/game_mode/nuclear)
+	
+/datum/uplink_item/device_tools/arm/nuke
+	name = "Additional Arm"
+	desc = "An additional arm, automatically added to your body upon purchase, allows you to use more items at once"
+	item = /obj/item/flashlight //doesn't actually spawn a flashlight, but it needs an object to show up in the menu :^)
+	cost = 15
```

Comment:
You don't have to copy over the entire datum definition. Just toss the new ``cost`` value here.